go-backend/db/user_blacklisted_tokens.go

```diff
@@ -17,3 +30,13 @@ func (s *pgStore) CleanBlacklistedTokens() (err error) {
 	}
 	return
 }
+
+func (s *pgStore) CreateUserBlacklistedToken(ctx context.Context, blacklistedToken UserBlacklistedToken) (err error) {
+	_, err = s.db.Exec(insertBlacklistedToken, blacklistedToken.UserID, blacklistedToken.Token, blacklistedToken.ExpirationDate)
+
+	if err != nil {
+		logger.WithField("err", err.Error()).Error("Error inserting the blacklisted token")
```

Comment:
Suggestion:

```go
logger.WithField("err", err.Error()).Error("Error inserting the blacklisted token for user with id " + blacklistedToken.UserID)
```

So our errors - and whatever ends up reporting them - have something useful to duplicate the problem in the log file. (It's ok if this is done in a future commit, imo.)

Reply:
Implemented in  fb35e6c